src/base/Ownable.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/**
+ * @dev Copy from openZeppelin contracts, and remove unnecessary functions.
```

Comment:
Can we also add the version or specific hash here in case future devs wanna reference ?

Reply:
Done, please approve again @chefburger